mmselfsup/models/algorithms/mae.py

```diff
@@ -30,24 +39,23 @@ def __init__(self, backbone=None, neck=None, head=None, init_cfg=None):
     def init_weights(self):
         super(MAE, self).init_weights()
 
-    def extract_feat(self, img):
+    def extract_feat(self, img: torch.Tensor) -> Tuple[torch.Tensor]:
         """Function to extract features from backbone.
 
         Args:
-            img (Tensor): Input images of shape (N, C, H, W).
-
+            img (torch.Tensor): Input images of shape (N, C, H, W).
         Returns:
-            tuple[Tensor]: backbone outputs.
+            tuple[torch.Tensor]: backbone outputs.
```

Comment:
```suggestion
            Tuple[torch.Tensor]: backbone outputs.
```
If there are items in current data type, better use the capital version. Please check other parts.